### ISSUE TITLE
Documentation browser is empty

### ISSUE THREAD

**Issue by GregoryTravis** [enso-org/enso#7623]
### What type of issue is this?

Permanent – Occurring repeatably

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

Creating a filter node from a table, I type "filter " and wait, nothing comes up in the docs browser.

### Expected behaviour

I expect to see the documentation for `Table.filter`.

### How we can reproduce it?

- Create a new enso project
- Drag csv files from 8 Aug book club into project
- Create a node containing just "filter " as a child of 'ee_dim_input.csv'

### Screenshots or screencasts

https://github.com/enso-org/enso/assets/24406074/b6a0314d-d9bb-465f-bb77-e1757455319f

### Logs

[2023-08-21T20-24-10.503Z-ide-2023.2.1-dev.log](https://github.com/enso-org/enso/files/12402386/2023-08-21T20-24-10.503Z-ide-2023.2.1-dev.log)

### Enso Version

Local build at 8e97f7a9e69440c8aeac7f9a1b92b07e31ccd1df

### Browser or standalone distribution

Standalone distribution (local project)

### Browser Version or standalone distribution

standalone

### Operating System

MacOS

### Operating System Version

Monterey 12.6.3

### Hardware you are using

MacBook Air (M1, 2020)

**Comment by farmaazon** [comment]
It is reproducible, but only if typed "filter " really fast. I was able to reproduce a similar issue only when I pasted the string immediately (my docs panel wasn't empty, but displayed docs for "at" function).

The cause: the actual mechanics is that we just display documentation of the selected entry. If no entry is selected, we do not change the docs.

When typing slowly, it looks good, because having typed "filter" will select "filter" method and display its docs. Then, when typed " ", we display a new list for an argument, and keep the old documentation (as no entry is selected).

A possible workaround for now is to not put space after function name to read its docs.
